### PR TITLE
Add hexo filter nofollow to plugin list & News

### PR DESCRIPTION
**Pull request by SukkaW**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [x] I want to publish my plugin on Hexo official website.
    - [x] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [x] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

News: hexo-filter-nofollow joined hexo official plugins

### CODE REVIEW

File: source/_posts/2019-09-10-hexo-filter-nofollow-joined-hexo-official-plugin.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+[hexo-filter-nofollow] will add `rel="external nofollow noreferrer"` to all external links automatically for security, privacy and SEO. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types).
+
+You can install this plugin using the command below:
+
+
+``` bash
+$ npm i hexo-filter-nofollow --save
+```
+
+And add configuration to your hexo's `_config.yml`:
```
Comment (curbengh) [inline source/_posts/2019-09-10-hexo-filter-nofollow-joined-hexo-official-plugin.md]: and enable it through your hexo's \`_config.yml\`:

Comment (SukkaW) [reply source/_posts/2019-09-10-hexo-filter-nofollow-joined-hexo-official-plugin.md]: Updated

**Comment by SukkaW** [comment]
Pending https://github.com/hexojs/hexo-filter-nofollow/pull/3 . After `hexo-filter-nofollow@2.0.2` published, the news article needs update as well.

**Comment by curbengh** [comment]
Pending https://github.com/hexojs/hexo-filter-nofollow/pull/2 and publish of v2.0.2.

**Comment by SukkaW** [comment]
https://github.com/hexojs/hexo-filter-nofollow/pull/4 is merged and 2.0.2 is published: https://www.npmjs.com/package/hexo-filter-nofollow

The article has been updated.

**Pull request merged**